app/src/main/kotlin/com/wire/android/ui/sharing/SendMessagesSnackbarMessages.kt

```diff
@@ -21,10 +21,9 @@ import com.wire.android.R
 import com.wire.android.model.SnackBarMessage
 import com.wire.android.util.ui.UIText
 
-sealed class ImportMediaSnackbarMessages(override val uiText: UIText) : SnackBarMessage {
-    object MaxImageSize : ImportMediaSnackbarMessages(UIText.StringResource(R.string.error_conversation_max_image_size_limit))
-    object MaxAmountOfAssetsReached :
-        ImportMediaSnackbarMessages(UIText.StringResource(R.string.error_limit_number_assets_imported_exceeded))
+sealed class SendMessagesSnackbarMessages(override val uiText: UIText) : SnackBarMessage {
+    data object MaxAmountOfAssetsReached : // TODO add max amount to string resource
```

Comment:
guess this `TODO` can be removed

Reply:
or a new string with proper `param`?

Reply:
I didn't want to change all strings translations to have more files changed